### ISSUE TITLE
Band changes move the frequency to the initial frequency of that band

### ISSUE THREAD

**Issue by ea5wa** [ea4k/klog#418]
Before changing the band, KLog should save the frequency in use and come to that frequency the next time this band is selected

**Comment by barjac** [comment]
In what mode? Seems rather an odd request to me. Surely the operator operates the radio and the logbook logs.the contacts. What am I missing? Does the rig not save the last frequency used on a band, mine does and it's quite old. With hamlib operating why would you need both rig and klog to remember the last frequency and which would win in the fight for control?

**Comment by ea5wa** [comment]
Yes, you are right. When using hamlib should be better ask to the rig in which frequency is, when changing band

**Comment by ea4k** [comment]
So, the process would be:
- The user selects is KLog a new band.
- KLog asks the radio to switch to that band without sending any freq?
I don't know if hamlib supports that usecase...but I will check
